test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -508,71 +508,68 @@ export default class TabularView extends Component {
                         newArray.push(dataObject);	
                     }
                 }
+                let benchmark = testResultsObject.aggregateInfo[aggregateIndex].benchmarkName;
+                let propers = await getBenchmarkMetricProps(benchmark);
+                if(propers){
+                    this.setState({allMetricsProps:{benchmark: propers}});
+                }
+                this.setState({allMetricsProps:{benchmark}});
```

Comment:
- we should not do it here for all benchmark. The value should be checked in `populateCompTable ()`
- In `populateCompTable ()`, as mentioned before, please do not use `state`. It is asynchronous. We may have issues in the loop. Just use a variable in constructor() to store this